include/awkward/Content.h

```diff
@@ -66,11 +65,11 @@ namespace awkward {
     virtual const std::shared_ptr<Content> getitem_next(const SliceField& field, const Slice& tail, const Index64& advanced) const;
     virtual const std::shared_ptr<Content> getitem_next(const SliceFields& fields, const Slice& tail, const Index64& advanced) const;
 
-    const std::shared_ptr<Content> getitem_next_array_wrap(const std::shared_ptr<Content> outcontent, const std::vector<int64_t>& shape) const;
+    const std::shared_ptr<Content> getitem_next_array_wrap(const std::shared_ptr<Content>& outcontent, const std::vector<int64_t>& shape) const;
 
   protected:
     std::shared_ptr<Identity> id_;
```

Comment:
@jpivarski - shouldn't this be const?

Reply:
Isn't it const? I see a const keyword here; the difference is in passing it by reference.